### PR TITLE
Quiz design QA

### PR DESCRIPTION
**Pull request by deadlybutter**
### What does this PR do?
- Changes font weight & size of the question
screen shot 2017-09-01 at 10 14 31 am

### CODE REVIEW

File: resources/assets/components/Quiz/quiz.scss [hunk 1]
```diff
@@ -9,8 +9,9 @@
 
   .quiz__title {
     color: $primary-color;
-    font-size: $font-large;
+    font-size: $font-hero;
```
Comment (weerd) [inline resources/assets/components/Quiz/quiz.scss]: Speaking of font-sizes... do we not want to have a smaller size for mobile and then increase it after a breakpoint? I saw on Ashley's phone yesterday when she was going through the quiz on mobile and the title's seemed a little "large" on mobile... or keep the size if that's preferred but reduce the padding on around it on mobile?

Comment (deadlybutter) [reply resources/assets/components/Quiz/quiz.scss]: Hm it looked fine for me but yeah we can break point this now that it jumped up a lot

**Comment by deadlybutter** [comment]
@weerd 👁

File: resources/assets/components/Quiz/quiz.scss [hunk 2]
```diff
@@ -33,8 +33,13 @@
       color: $white;
       font-family: $secondary-font-family;
       font-size: $font-larger;
+      font-weight: normal;
       letter-spacing: 2px;
       padding: $base-spacing;
```
Comment (weerd) [inline resources/assets/components/Quiz/quiz.scss]: maybe `$half-spacing` for small screen and `$base-spacing` for the medium breakpoint?

image

Looks like a lot of spacing around question header on small screen. Thoughts?

Comment (deadlybutter) [reply resources/assets/components/Quiz/quiz.scss]: or actually I can just remove the title padding on mobile

Comment (deadlybutter) [reply resources/assets/components/Quiz/quiz.scss]: screen shot 2017-09-01 at 10 42 11 am

**Pull request merged**